src/man/include/ad_modified_defaults.xml

```diff
@@ -92,6 +92,18 @@
                     this fallback behavior, you can explicitly
                     set "fallback_homedir = %o".
                 </para>
+                <para>
+                    Note that the system typically expects home directory
+                    in /home/%u folder. If you decide to use a different
+                    directory structure, some other parts of your system may
+                    need adjustments.
+                </para>
+                <para>
+                    For example automated creation of home directories in
+                    combination with selinux requires selinux adjustment,
+                    otherways home directory will be created with wrong
```

Comment:
I think you meant "otherwise" instead of "otherways".

I'd change to:
"otherwise the home directory will be"